apps/src/applab/applab.js

```diff
@@ -1247,6 +1245,8 @@ Applab.runButtonClick = function() {
   }
   studioApp.reset(false);
   studioApp.attempts++;
+  clearDebugOutput();
+  clearDebugInput();
```

Comment:
_Probably_ not an issue, but I've wondered in studio as well - should this code live in `Applab.execute()` in case something bypasses `runButtonClick`?